src/go/rpk/pkg/cli/transform/list.go

```diff
@@ -147,9 +151,9 @@ func printDetailed(f config.OutFormatter, d []detailedTransformMetadata, w io.Wr
 		}
 		tw.Print(fmt.Sprintf("%s, %s → %s", m.Name, m.InputTopic, strings.Join(m.OutputTopics, ", ")))
 		// add an empty column to provide an indent
-		tw.Print("", "PARTITION", "NODE", "STATUS")
+		tw.Print("", "PARTITION", "NODE", "STATUS", "LAG")
```

Comment:
[Suggestion, not necessarily for this PR]: Maybe it's worth adding a longer help text to this command explaining what's in the table, the possible outcomes of status, and the meaning of lag in this context.

So far it's just running, inactive, and errored, but maybe there is a way to check the error or something like that.

Reply:
Oh yes, thank you for this suggestion. I'll do this in a followup